tasks/e2e-background-jobs/run.mts

```diff
@@ -546,17 +536,10 @@ async function runCronJob(projectPath: string) {
   console.log(`           It is now ${now} (delta: ${delta}ms)`)
 
   try {
-    const jobsProcess = $`yarn rw jobs work`.nothrow().quiet()
-
-    const jobsProcessPid = jobsProcess.pid
-    if (!jobsProcessPid) {
-      throw new Error('jobsProcessPid is undefined')
-    }
-
-    // 3600 was enough of a timeout locally, but I had to increase it for CI
-    setTimeout(() => kill(jobsProcessPid, 'SIGKILL'), 9600)
-
-    const { stdout, stderr } = await jobsProcess
+    const { stdout, stderr } = await $`yarn rw jobs work`
+      .nothrow()
+      .quiet()
+      .timeout(9600)
```

Comment:
Hmm... zx uses `SIGTERM` by default. Let's check if this is significant.
`$.timeoutSignal = 'SIGKILL'` or `task.timeout(9600, 'SIGKILL')`